contrib/docs/tutorials/aws-alb-aad-oidc-auth.md

```diff
@@ -154,16 +154,16 @@ export default async function createPlugin({
             const [id] = email?.split('@') ?? '';
             // Fetch from an external system that returns entity claims like:
             // ['user:default/breanna.davison', ...]
-            const ent = [`user:default/${id}`];
+            const userEntityRef = `user:default/${id}`;
```

Comment:
Could you make this `stringifyEntityRef({ kind: 'User', namespace: DEFAULT_NAMESPACE, name: id })` instead? That takes care of normalization (uses the right case rules) etc, which is something that can bite people if it's wrong

Reply:
@freben I've included your suggestion, thanks!